### DISCUSSION TITLE
24 Hour timeformat

### DISCUSSION THREAD

**Discussion by JunChatani** [airbnb/visx#1150] [Q&A] [answered]
Hi Team,

Newbie here, but I was wondering how I can change the timeformat from AM/PM to a 24hr timeformat for the Axis. 

I am not very familiar with d3 yet, but saw this could be achieved by using something like timeFormat("%H %M") in vanilla d3. 
I also saw in VX examples that the formatter can be used for tooltips, but how do I do this easily for the Axis in Visx?

Thanks in advance!

**Comment by williaster** [comment, marked as answer]
Hey @JunChatani 👋  thanks for checking out `visx`. I think what you're looking for here is the `@visx/axis` `tickFormat` prop (see the [docs](https://airbnb.io/visx/docs/axis#Axis_tickFormat)).

In the [`axis` demo](https://airbnb.io/visx/axis), you can see that we use this to change how we format a specific tick. In your case you would use the function to format your `Date` (or `string` or `number` depending on your raw data format for the `Axis`).

<img src="https://user-images.githubusercontent.com/4496521/114088866-5716cf00-986a-11eb-9e86-afedf2145985.png" width="600" />

Hope that helps!

**Reply by JunChatani** [reply]
Thank you for the quick reply, I completely overlooked that part, my bad. 
I managed to do it with that part, appreciate it!
